lingua_franca/res/countries.json

```diff
@@ -0,0 +1,3227 @@
+        "official_name_ar": "ألبانيا",
+        "official_name_es": "Albania",
+        "official_name_cn": "阿尔巴尼亚",
+        "official_name_en": "Albania",
+        "official_name_ru": "Албания",
+        "Language": "sq"
+    },
+    {
+        "ISO3166-1-Alpha-3": "DZA",
+        "ISO3166-1-Alpha-2": "DZ",
+        "official_name_fr": "Algérie",
+        "ISO4217-currency_name": "Algerian Dinar",
+        "ISO4217-currency_alphabetic_code": "DZD",
+        "official_name_ar": "الجزائر",
+        "official_name_es": "Argelia",
+        "official_name_cn": "阿尔及利亚",
+        "official_name_en": "Algeria",
+        "official_name_ru": "Алжир",
+        "Language": "ar-DZ"
+    },
+    {
+        "ISO3166-1-Alpha-3": "ASM",
+        "ISO3166-1-Alpha-2": "AS",
+        "official_name_fr": "Samoa américaines",
+        "ISO4217-currency_name": "US Dollar",
+        "ISO4217-currency_alphabetic_code": "USD",
+        "official_name_ar": "ساموا الأمريكية",
+        "official_name_es": "Samoa Americana",
+        "official_name_cn": "美属萨摩亚",
+        "official_name_en": "American Samoa",
+        "official_name_ru": "Американское Самоа",
+        "Language": "en-AS"
+    },
+    {
+        "ISO3166-1-Alpha-3": "AND",
+        "ISO3166-1-Alpha-2": "AD",
+        "official_name_fr": "Andorre",
+        "ISO4217-currency_name": "Euro",
+        "ISO4217-currency_alphabetic_code": "EUR",
+        "official_name_ar": "أندورا",
+        "official_name_es": "Andorra",
+        "official_name_cn": "安道尔",
+        "official_name_en": "Andorra",
+        "official_name_ru": "Андорра",
+        "Language": "ca"
+    },
+    {
+        "ISO3166-1-Alpha-3": "AGO",
+        "ISO3166-1-Alpha-2": "AO",
+        "official_name_fr": "Angola",
+        "ISO4217-currency_name": "Kwanza",
+        "ISO4217-currency_alphabetic_code": "AOA",
+        "official_name_ar": "أنغولا",
+        "official_name_es": "Angola",
+        "official_name_cn": "安哥拉",
+        "official_name_en": "Angola",
+        "official_name_ru": "Ангола",
+        "Language": "pt-AO"
+    },
+    {
+        "ISO3166-1-Alpha-3": "AIA",
+        "ISO3166-1-Alpha-2": "AI",
+        "official_name_fr": "Anguilla",
+        "ISO4217-currency_name": "East Caribbean Dollar",
+        "ISO4217-currency_alphabetic_code": "XCD",
+        "official_name_ar": "أنغيلا",
+        "official_name_es": "Anguila",
+        "official_name_cn": "安圭拉",
+        "official_name_en": "Anguilla",
+        "official_name_ru": "Ангилья",
+        "Language": "en-AI"
+    },
+    {
+        "ISO3166-1-Alpha-3": "ATA",
+        "ISO3166-1-Alpha-2": "AQ",
+        "official_name_fr": "Antarctique",
+        "ISO4217-currency_name": "No universal currency",
+        "ISO4217-currency_alphabetic_code": "",
+        "official_name_ar": "أنتاركتيكا",
+        "official_name_es": "Antártida",
+        "official_name_cn": "南极洲",
+        "official_name_en": "Antarctica",
+        "official_name_ru": "Антарктике"
```

Comment:
No language.. should it be `None` or an empty string to prevent reference errors?

Reply:
this simply said "world" in source data

not sure if adding empty string or None makes sense... are we making a distinction between undefined/unknown?

i usually look at this stuff as responsibility of whatever is reading the file, data provides datapoints about stufff it knows about, not about what is missing

Reply:
Hmm. I guess this will be up to LF to handle, so I'm not sure what will be easier to parse.. I think an empty value will make it more obvious at a glance if something is missing, though I'm not sure it makes much difference unless there's an instance where there is no language (I don't think that should ever be the case?). Empty string would probably be the easiest way to specify the language is known to be none, since we could just use a type check to validate (`if isinstance(lang, str) -> validated lang is specified`)